Core/FunctionTree/FunctionTreeEstimatorWrapper.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef CORE_FUNCTIONTREEESTIMATORWRAPPER_HPP_
+#define CORE_FUNCTIONTREEESTIMATORWRAPPER_HPP_
+
+#include <memory>
+
+#include "Estimator/Estimator.hpp"
+#include "ParameterList.hpp"
+
+namespace ComPWA {
+namespace FunctionTree {
+
+class FunctionTree;
+
+class FunctionTreeEstimatorWrapper
+    : public ComPWA::Estimator::Estimator<double> {
+public:
+  FunctionTreeEstimatorWrapper(std::shared_ptr<FunctionTree> tree,
+                               ParameterList parameters);
+  double evaluate();
+  void updateParametersFrom(const std::vector<double> &params);
+  std::vector<double> getParameters() const;
+
+  std::shared_ptr<FunctionTree> getFunctionTree() { return Tree; }
+  ParameterList &getParameterList() { return Parameters; };
+  ParameterList &getUserParameterList() { return UserParameters; };
+
+  std::string print(int level) const;
+
+private:
+  std::shared_ptr<FunctionTree> Tree;
+  ParameterList Parameters;
+  ParameterList UserParameters;
```

Comment:
What is the difference between `Parameters` and `UserParameters`?

Reply:
UserParameters save the state of the fit parameters when read from the xml (so default values as stated in the xml). Its a workaround to the following problem. The problem is, that I have to reset all fit parameters to NOT fixed, so that I satisfy the new Intensity/Estimator interface.